php-practice/section5.php

```diff
@@ -114,7 +116,11 @@ function getSmallestValue($arr) {
  *
  * Explain in your own words how the above function works.
  *
- * ANSWER:
+ * ANSWER: first we name our function (in this case we give it the name getSmallestValue) followed by the parameters we want to pass.
+ * Inside the function, we first define what we want the function to do. In this case we want it to sort the given array. We use the PHp array sort function "sort"
+ * followed by the array we want it to sort "$arr". We then tell the function to "return $arr[0];" which is the value we want. We then print out that value
+ * using the "echo" keyword followed by the function name "getSmallestValue" then in () the array we want sorted. To make the printed value easier to read
+ * we concatenate the array with PHP_EOL so it prints the sorted value on another line.
```

Comment:
Good!

Can you explain why we return the first element of the array (`$arr[0]`) instead of any others?